src/util/helpHelpers.js

```diff
@@ -28,27 +23,30 @@ const formatCommands = (command) => {
   return formattedStr;
 };
 
-const formatOption = option => (
-  `${indentFurther}${getStyledText(option.name, 'green')}: ${option.description}`
-);
+const formatOption = (option) =>
+  `${indentFurther}${getStyledText(option.name, 'green')}: ${
+    option.description
+  }`;
```

Comment:
This is also interesting. Is the template string split across two lines because of max line length?